### DISCUSSION TITLE
cookie related pop-ups

### DISCUSSION THREAD

**Discussion by mpaunovic** [bromite/bromite#1813] [Q&A]
Hey guys, I was wondering how do you get rid of all the tedious cookie warnings/consents bombarding us every day ? There are specialized filter lists (like Easylist Cookie List) for uBO but they're not implemented in Bromite.

As far as I know @xarantolus is maintaining unofficial lists and scripts that can help...

https://github.com/xarantolus/filtrite#lists

https://github.com/xarantolus/bromite-userscripts/#bromite-user-scripts

Feel free to share any other solution that you know of. :)

**Comment by Universalizer** [comment]
https://github.com/bromite/bromite#filters-credits, Helpful.
